test/e2e/network/service.go

```diff
@@ -3248,8 +3247,7 @@ var _ = SIGDescribe("ESIPP [Slow]", func() {
 		}
 	})
 
-	// TODO: Get rid of [DisabledForLargeClusters] tag when issue #90047 is fixed.
-	ginkgo.It("should handle updates to ExternalTrafficPolicy field [DisabledForLargeClusters]", func() {
+	ginkgo.It("should handle updates to ExternalTrafficPolicy field", func() {
```

Comment:
Please don't mix removing the "DisabledForLargeCluster" tag together with changes of the test.
We should enable it independently from the chances to the test.

Reply:
oh I thought this test was skipped because of this tag, will add it back.

Reply:
fixed

Reply:
It is skipped because of this tag. But this is conscious - I don't want to enable it togehter with the changes.